src/js/test/supervisors/FileManager.test.ts

```diff
@@ -209,13 +209,13 @@ describe("FileManager", () => {
       "on enable request",
     function (done) {
       const handle = createHandle();
-      const repo = new Repository();
-      const removeSpy = sinonInstance.spy(repo, "remove");
       const {
         moveCoprocessor,
         getCoprocessor,
         enableCoprocessor,
       } = createStubs(sinonInstance);
+      const repo = new Repository();
+      const removeSpy = sinonInstance.spy(repo, "remove");
```

Comment:
These changes seem kind of arbitrary. Do they have any effect or are they purely cosmetic?

Reply:
this change isn't cosmetic, I move the creation `Repository` before creating stubs for that class, before this change there isn't a necessity to stub anything  for that reason it was before `createStubs`

Reply:
Do you mean "after"?

Reply:
Sorry, right, I move it after creates stubs. in order to mock another function before creating Repository instance.